mypy/pyinfo.py

```diff
@@ -24,12 +24,16 @@ def getprefixes():
 
 def getsitepackages():
     # type: () -> List[str]
-    if hasattr(site, 'getusersitepackages') and hasattr(site, 'getsitepackages'):
-        user_dir = site.getusersitepackages()
-        return site.getsitepackages() + [user_dir]
+    res = []
+    if  hasattr(site, 'getsitepackages'):
```

Comment:
Looks like you have a stray space here :/
```suggestion
    if hasattr(site, 'getsitepackages'):
```

Reply:
yup, sorry about that